pyfstat/core.py

```diff
@@ -993,10 +1008,22 @@ def get_fullycoherent_detstat(
         tstart=None,
         tend=None,
     ):
-        """Computes the detection statistic (twoF or log10BSGL) fully-coherently at a single point.
+        """Computes the detection statistic fully-coherently at a single point.
+
+        Currently supported statistics:
+
+        * twoF (CW)
+
+        * log10BSGL (CW or transient)
+
+        * maxTwoF (transient)
+
+        * logBtSG (transient)
```

Comment:
```suggestion
        * log10BtSG (transient)
```
If not, why? Is `log10BSGL` historical and `logBtSG` ye new trend? In such case, `log` ---> `ln`.

Reply:
Historical divergence in lalsuite conventions for the two statistics. I thought about normalising to log10, but then one always has to convert when wanting to compare to lalsuite outputs. Using the less ambiguous `ln` is a very good suggestion, though.